lib/modulesync.rb

```diff
@@ -80,7 +82,7 @@ def self.run(args)
         files_to_manage -= files_to_delete
         if options[:noop]
           Git.update_noop(puppet_module, options)
-        else
+        elsif not options[:offline]
```

Comment:
I don't think this addresses https://github.com/puppet-community/modulesync/pull/53#discussion_r36565658

I think line 83 should be if options[:noop] || options[:offline] since we want the same behavior (printing out the changes without making updates)